crates/autopilot/src/run_loop.rs

```diff
@@ -39,9 +38,9 @@ use {
 pub struct RunLoop {
     pub eth: infra::Ethereum,
     pub persistence: infra::Persistence,
+    pub drivers: Vec<infra::Driver>,
```

Comment:
should i rename this to `solvers: Vec<infra::Solver`? Do we want to refer to driver as `Solver` in the future?

Reply:
Renaming makes sense to me. But I think I would only do it after the `autopilot` was completely DDD-ified.

Reply:
I'd also support this renaming

Reply:
Ok, let's keep the renaming for later. Anyway it will be trivial.